Tutorials/Animations_with_xmovie.ipynb

```diff
@@ -0,0 +1,1515 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        ax.set_title(f"time: {str(da['time'][timestamp].values)}")
</pre><p>Could get rid of a bunch of zeros in the title? e.g. </p><p><code>ax.set_title(f"time: {str(da['time'][timestamp].values)[:-19]}")</code></p><p>Though better if there was a nicer way to do that than arbitrary indexing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/452/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
True, I didn't bother I admit. I thought it goes beyond the aim of the tutorial. I don't want to hardcode `[:-19]` which might work for this case but not for something else....

Reply:
I'll add an issue for this and we can tackle in the future.